hallux/targets/github_proposal.py

```diff
@@ -120,9 +122,11 @@ def commit_diff(self) -> bool:
                 side="RIGHT",
                 path=str(compacted.filename),
                 line=compacted.end_line,
-                start_line=compacted.start_line,
+                start_line=compacted.start_line if compacted.start_line < compacted.end_line else GithubObject.NotSet,
+                start_side="RIGHT" if compacted.start_line < compacted.end_line else GithubObject.NotSet,
             )
-        except GithubException:
+        except BaseException as ex:
```

Comment:
Catch a more specific exception or reraise the exception
```suggestion
        except SpecificException as ex:  # Replace BaseException with the specific exception class
```